### PR TITLE
fix eventing/samples/helloworld-python

### LINKED ISSUE #3679: Hello World - Python(docs/eventing/samples/helloworld/helloworld-python/_index.md)
## Expected Behavior
The sample to create a Broker also.

## Actual Behavior
The yaml does not create a Broker.

## Steps to Reproduce the Problem

1.
1.
1.

## Additional Info
**Additional context**
https://stackoverflow.com/questions/67720835/knative-eventing-newobservedgenfailure-error-for-helloworld-samples-trigger/67722535#67722535

**Install information:**
 - Platform (GKE, IKS, AKS, etc.):
 - Knative Version:

### PR DESCRIPTION
**Pull request by novahe**
Fixes #3679

## Proposed Changes 
-  The sample to create a Broker also.
- Reformat YAML

### CODE REVIEW

**Comment by abrennan89** [comment]
/approve

I'll let @vaikas double check and lgtm this since he raised the issue. Thanks @novahe !

File: docs/eventing/samples/helloworld/helloworld-python/README.md [hunk 1]
```diff
@@ -97,7 +97,15 @@ cd knative-docs/docs/eventing/samples/helloworld/helloworld-python
       labels:
            eventing.knative.dev/injection: enabled
     ---
-    # Helloworld-python app deploment
+    # A default broker
+    apiVersion: eventing.knative.dev/v1
+    kind: Broker
+    metadata:
+      name: default
+      namespace: knative-samples
```
Comment (vaikas) [inline docs/eventing/samples/helloworld/helloworld-python/README.md:105]: We also need to add a kind of broker we want to create. So, something like:
```
name: default
namespace: knative-samples
annotations:
  eventing.knative.dev/broker.class: MTChannelBasedBroker
```
And maybe add a note saying that if you are using different kinds of brokers you might have to change this?

Comment (novahe) [reply docs/eventing/samples/helloworld/helloworld-python/README.md:105]: done! PTAL

**Review by csantanapr — CHANGES_REQUESTED** [review]
Thank you for helping on this issue and the contribution 

Found a few problems in both go and python samples

1. Link to section send event doesn't work on both
Replace
```
1. [Send a CloudEvent to the Broker](###Send-CloudEvent-to-the-Broker)
```
With
```
1. [Send a CloudEvent to the Broker](#send-and-verify-cloudevents)
```

2. URL for the broker in python sample is wrong
Replace
```
curl -v "default-broker.knative-samples.svc.cluster.local"
```
with
```
curl -v "broker-ingress.knative-eventing.svc.cluster.local/knative-samples/default"
```

3. Event Ce-Type is wrong in Go sample
Replace
```
-H "Ce-Type: dev.knative.samples.helloworld"
```
With:
```
-H "Ce-Type: dev.knative.samples.hifromknative"
```

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Pull request merged**